packages/codemods/src/codeMods/__tests__/mock/compat/Button.tsx

```diff
@@ -0,0 +1,2 @@
+export const Button = {};
```

Comment:
Not sure if it would work in this case, but you should consider following the automatic mocking folder structure described here. https://jestjs.io/docs/en/manual-mocks.html

Reply:
It probably won't since we just care the files exist to be parsed into typescript asts.